### PR TITLE
Add Footer component

### LINKED ISSUE #206: Implement Footer

### PR DESCRIPTION
**Pull request by joneskj55**
### What does it do?
Adds new footer component

### Any helpful background information?
[Discord Chat](https://discord.com/channels/883478451850473483/884852584941232169/946474631760601098)

### Any new dependencies? Why were they added?
No

### Relevant screenshots/gifs
Screen Shot 2022-02-28 at 11 39 16 AM

Screen Shot 2022-02-28 at 11 39 23 AM

Screen Shot 2022-02-25 at 4 14 13 PM

Screen Shot 2022-02-25 at 4 14 27 PM

### Does it close any issues?

Closes #206

### CODE REVIEW

File: src/Components/Footer/index.tsx [hunk 1]
```diff
@@ -1,7 +1,149 @@
...
+    </Text>
+  );
+};
 
 const Footer = () => {
-  return <Box>Footer</Box>;
+  const { colorMode } = useColorMode();
+  return (
+    <Box>
+      <Container as={Stack} maxW={'6xl'} py={10}>
```
Comment (Nazeeh21) [inline src/Components/Footer/index.tsx:28]: `Footer` should occupy the full width of the website.
I've already added `paddingY` in the `Layout` component, so it don't require adding more padding to the components

image

Comment (Nazeeh21) [inline src/Components/Footer/index.tsx:28]: I think removing this `Container` will fix this

**Review by Nazeeh21 — CHANGES_REQUESTED** [review]
`Footer` should occupy the full width of the website.
I've already added `paddingY` in the `Layout` component, so it don't require adding more padding to the components

image

Also, the responsiveness of the component needs to be approved, right now it doesn't seem responsive for screen sizes less than 800px 

image

**Comment by Nazeeh21** [comment]
Hey @joneskj55, just tag me here when you feel PR is ready for review, thanks

File: src/Components/Footer/index.tsx (hunk 1 above)
Comment (joneskj55) [reply src/Components/Footer/index.tsx]: Footer now spans full width

Comment (joneskj55) [reply src/Components/Footer/index.tsx]: 👍🏾

**Comment by joneskj55** [comment]
Hi, @Nazeeh21 please review at your convenience. Thanks.

**Comment by Nazeeh21** [comment]
Thanks @joneskj55, will review it soon 😊

**Comment by joneskj55** [comment]
@Nazeeh21 Awesome, and thanks for the assist as well!

**Comment by Nazeeh21** [comment]
https://user-images.githubusercontent.com/56908732/159897800-14851102-6b4a-4c53-b84a-c3f37707bd78.mov

@joneskj55 I think we need to work on reducing the gap between the **useful links**, **discover** and **social** as on screen size between 750 to 1074, D_D heading and **useful links** is getting overlapped. You can see it in the video I attached

**Comment by joneskj55** [comment]
Aahh good catch! @Nazeeh21 I'll get on this ASAP.

**Comment by joneskj55** [comment]
Uploading Screen Recording 2022-03-26 at 1.39.16 PM.mov…

@Nazeeh21 I think that Vercel deployment may be a commit behind. I haven't made any changes and running the project locally I am getting what's shown in this attached video.

**Comment by Nazeeh21** [comment]
Ohh is it? I'll test it out tomorrow locally and everything works fine, we'll just merge it once we authorize the vercel. Thanks for your patience

**Comment by Nazeeh21** [comment]
@joneskj55 I tested out the PR locally and it's amazing and I think ready to be merged, once we've vercel deployment to these.

**Pull request merged**